src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +45,119 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    Query myQuery = new Query("Initiative");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery myResults = datastore.prepare(myQuery);
+    List<String> titles = new ArrayList<>();
+    List<String> locations = new ArrayList<>();
+    List<String> startDates = new ArrayList<>();
+    List<String> endDates = new ArrayList<>();
+    List<String> contexts = new ArrayList<>();
+    List<String> causes = new ArrayList<>();
+    List<String> audiences = new ArrayList<>();
+    List<String> instructions = new ArrayList<>();
+    List<String> urls = new ArrayList<>();
+    List<List<String>> listOfLists = new ArrayList<>();
```

Comment:
Instead of storing each field separately into different lists, I recommend storing all the fields into a class object Initiative and just have a list of Initiative objects. This way it is guaranteed that all fields associated with one initiative are stored together since list as a structure doesn't maintain ordering.

Reply:
The thing about storing all of these fields as an object is that sometimes I want to reference just specific parts of the object and going through all of the objects looking for those specific parts would take more time and work, in my opinion. This is why I made it in separate lists.